<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BG83L86" failures="1" tests="5" name="com.bank.api.tests.UserProfileManagementTests" time="2.950" errors="0" timestamp="2025-02-07T14:25:49 IST" skipped="0">
  <testcase classname="com.bank.api.tests.UserProfileManagementTests" name="verifyGetUserProfile" time="0.555"/>
  <testcase classname="com.bank.api.tests.UserProfileManagementTests" name="verifyUpdateUserProfileWithPutRequest" time="0.518"/>
  <testcase classname="com.bank.api.tests.UserProfileManagementTests" name="verifyUpdateUserProfileWithPatchRequest" time="0.478"/>
  <testcase classname="com.bank.api.tests.UserProfileManagementTests" name="verifyChangePassword" time="0.861"/>
  <testcase classname="com.bank.api.tests.UserProfileManagementTests" name="verifyDeleteUserProfile" time="0.538">
    <failure message="expected [User profile deleted successfully] but found [{
    &quot;timestamp&quot;: &quot;2025-02-07 08:55:45&quot;,
    &quot;status&quot;: 500,
    &quot;error&quot;: &quot;System Error&quot;,
    &quot;message&quot;: &quot;An unexpected error occurred&quot;,
    &quot;path&quot;: &quot;uri=/api/users/profile&quot;,
    &quot;solution&quot;: &quot;Please try again later or contact support&quot;,
    &quot;errorCode&quot;: &quot;SYS_001&quot;
}]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [User profile deleted successfully] but found [{
    "timestamp": "2025-02-07 08:55:45",
    "status": 500,
    "error": "System Error",
    "message": "An unexpected error occurred",
    "path": "uri=/api/users/profile",
    "solution": "Please try again later or contact support",
    "errorCode": "SYS_001"
}]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.bank.api.tests.UserProfileManagementTests.verifyDeleteUserProfile(UserProfileManagementTests.java:143)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
  </testcase> <!-- verifyDeleteUserProfile -->
</testsuite> <!-- com.bank.api.tests.UserProfileManagementTests -->
